.cursor/rules/README.md

```
@@ -35,6 +35,11 @@ This directory contains comprehensive development rules and guidelines for the E
 |------|-------------|-------------|
 | [Translation Implementation](translation-implementation.mdc) | German 'Du' form requirements and component-based organization | Working with translations, i18n, or localization |
 
+### 🌐 Browser & UI verification
+| Rule | Description | Applied When |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)